src/common/Storage.tsx

```diff
@@ -0,0 +1,59 @@
+import moment from 'moment';
+import { AccountTokens } from "./types/Addresses";
+
+export function storeTokens(tokens: AccountTokens) {
+    const storage = getStorage();
+    clearTokens();
+    for(const address of Object.keys(tokens)) {
+        storage.setItem(TOKEN_KEY_PREFIX + address, JSON.stringify(tokens[address]));
+    }
+}
+
+function getStorage(): Storage {
+    return window.localStorage;
+}
+
+export function clearTokens() {
+    const toRemove: string[] = [];
+    forEachStorageKey(key => {
+        if(key!.startsWith(TOKEN_KEY_PREFIX)) {
+            toRemove.push(key!);
+        }
+    });
+
+    const storage = getStorage();
+    toRemove.forEach(key => storage.removeItem(key));
```

Comment:
Why are those 2 steps necessary ?

Reply:
To prevent any misbehavior caused by a modification of the collection during iteration.